blueprints/ember-frost-object-browser/index.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  description: '',
+  normalizeEntityName: function () {},
+
+  /**
+    Installs specified packages at the root level of the application.
+    Triggered by 'ember install <addon name>'.
+
+    @returns {Promise} package names and versions
+  */
+  afterInstall: function () {
+    return this.addAddonsToProject({
+      packages: [
+        {name: 'ember-frost-theme', target: '^1.3.2'},
+        {name: 'ember-frost-bunsen', target: '^2.2.0'},
+        {name: 'ember-frost-button', target: '^1.4.2'},
+        {name: 'ember-frost-checkbox', target: '^1.2.0'},
```

Comment:
Where is checkbox used?  If it's just in the form then it should come from bunsen